.travis.yml

```diff
@@ -2,6 +2,9 @@ language: python
 python:
     - "3.5"
     - "3.4"
+before_install:
+    - git clone https://github.com/ONSdigital/sdx-common.git
+    - pip3 install ./sdx-common
 install:
     - make
```

Comment:
Just spotted this as well, I pretty much copied the collect changes and didnt realise this was running slightly differently, next commit should have things sorted